pkg/tls/testutils/testutils.go

```diff
@@ -107,4 +107,40 @@ ksLi4xaNmjICq44Y3ekQEe5+NauQrz4wlHrQMz2nZQ/1/I6eYs9HRCwBXbsdtTLS
 R9I4LtD+gdwyah617jzV/OeBHRnDJELqYzmp
 -----END CERTIFICATE-----
 `
+	TLSAuthKey = `-----BEGIN EC PARAMETERS-----
```

Comment:
is this part of this PR?

Reply:
I thought I had all the TLS parts removed... will check